test/scripts/initialize_eks_cluster.bash

```diff
@@ -0,0 +1,45 @@
+LD_FLAGS=$1
+IMAGE_VERSION=$2
+
+AWS_REGION="us-west-2"
+AWS_ACCOUNT_ID=$(aws --region $AWS_REGION sts get-caller-identity --query Account --output text)
+
+ECR_REGISTRY_NAME="csisecretdriver"
+ECR_REGISTRY_URL="$AWS_ACCOUNT_ID.dkr.ecr.$AWS_REGION.amazonaws.com"
+ECR_REGISTRY_URL_WITH_NAME="$ECR_REGISTRY_URL/$ECR_REGISTRY_NAME"
+
+ECR_IMAGE_NAME="csisecretdriver"
+NAMESPACE="kube-system"
+EKS_CLUSTER_NAME="integ-cluster"
+AWS_SERVICE_ACCOUNT_NAME="basic-test-mount-sa"
+
+docker build  --no-cache --build-arg LDFLAGS="$LD_FLAGS" -t $ECR_IMAGE_NAME -f ./docker/Dockerfile .
```

Comment:
@aramase would it be possible for them to just add `make e2e-container` to test run instead of repeating build logic here?

Reply:
Yeah, with a custom registry just `make e2e-container` should be sufficient. Before running the `make e2e-container` command there should be a step to login to the container registry for AWS. Since we're not running test with windows, just `make container-linux` with custom registry should be enough.

For instance, `REGISTRY=aramase IMAGE_VERSION=v0.0.22-rc.1 make container-linux`

Reply:
I refactor this to avoid duplicating build logic